staging/dtm-jenkins-pipeline-example/springboot/Jenkinsfile

```diff
@@ -0,0 +1,38 @@
+version="1.0.0"
+repository="steinsliber/maven_test"
```

Comment:
How about to use "devstream/sprintboot-test" or some other more general name here?